src/assets/usage-examples-code/input/key_typed/key_typed-1-type-key.txt

```diff
@@ -0,0 +1,3 @@
+### key_typed() usage exaple
+
+The following code is an example of using [key typed](https://splashkit.io/api/input/#key-typed) to move a sprite in a game window to the right after right key is typed.
```

Comment:
```suggestion
The following code is an example of using [`key_typed`](https://splashkit.io/api/input/#key-typed) to move a sprite in a game window to the right after right key is typed.
```

Should probably run this past @omckeon, however my preference would be to have the function name in snake_case like how it is in the code and to wrap it in a code block. Same could be done for the page title.